### ISSUE TITLE
Try using custom_callbacks. Counting accuracy takes a lot of time, so don't use it every epoch.

### ISSUE THREAD

**Issue by t00lbelt** [matterport/Mask_RCNN#2674]
Try using custom_callbacks. Counting accuracy takes a lot of time, so don't use it every epoch.

mean_average_precision_callback = modellib.MeanAveragePrecisionCallback(model,\
          model_inference, dataset_val, calculate_map_at_every_X_epoch=5, verbose=1)

model.train(dataset_train, dataset_val, 
            learning_rate=config.LEARNING_RATE, 
            epochs=100,
            layers='heads',
            custom_callbacks=[mean_average_precision_callback])

_Originally posted by @VtlNmnk in https://github.com/matterport/Mask_RCNN/issues/1839#issuecomment-549430332_

**Comment by t00lbelt** [comment]
Hi Everyone,
I am new to neural networks and humbly ask for assistance. I am working on the matterport Mask-RCNN with a custom dataset. To calculate accuracy, I added the code recommended in the post to my training and model scripts. When I ran my training script, I received the following error:
![image](https://user-images.githubusercontent.com/35847967/130271558-ed4858a0-5d5e-48e8-a022-2430077ec410.png)
Where and how is 'model_inference' defined?
